test/service/test_repository_proxy.py

```diff
@@ -58,8 +82,11 @@ def setUpModule():
 mock_secret_access_key = 'test-secret-key'  # @mock_sts uses wJalrXUtnFEMI/K7MDENG/bPxRfiCYzEXAMPLEKEY
 mock_session_token = 'test-session-token'  # @mock_sts token starts with  AQoEXAMPLEH4aoAH0gNCAPyJxz4BlCFFxWNE1OPTgk …
 
+mock_tdr_service_url = f'https://serpentine.datarepo-dev.broadinstitute.net.test.{config.domain_name}'
+mock_tdr_sources = 'tdr:mock:snapshot/2021'
```

Comment:
Nitpick, but "2021" feels a little obtuse for a snapshot name here. Maybe just use "mock_snapshot" or something similarly explicit?

Reply:
Makes sense!